app/DataTables/ActivityLogDataTable.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\DataTables;
+
+use App\User;
+use Yajra\DataTables\Services\DataTable;
+use Spatie\Activitylog\Models\Activity;
+
+class ActivityLogDataTable extends DataTable
+{
+    /**
+     * Build DataTable class.
+     *
+     * @param mixed $query Results from query() method.
+     * @return \Yajra\DataTables\DataTableAbstract
+     */
+    public function dataTable($query)
+    {
+        return datatables($query);
+            //->addColumn('action', 'activitylog.action');
+    }
+
+    /**
+     * Get query source of dataTable.
+     *
+     * @param \App\User $model
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function query(User $model)
+    {
+        //return $model->newQuery()->select($this->getColumns());
+        $query = Activity::all();
+        
+        return $this->applyScopes($query);
+    }
+
+    /**
+     * Optional method if you want to use html builder.
+     *
+     * @return \Yajra\DataTables\Html\Builder
+     */
+    public function html()
+    {
+        return $this->builder()
+                    ->columns($this->getColumns())
+                    ->minifiedAjax()
+                    //->addAction(['width' => '80px'])
+                    ->parameters($this->getBuilderParameters());
+    }
+
+    /**
+     * Get columns.
+     *
+     * @return array
+     */
+    protected function getColumns()
+    {
+        return [
+            'id',
+            'log_name',
+            'description',
+            'subject_id',
+            'subject_type',	
+            'causer_id',
+            'causer_type',
+            'properties',
+            'created_at', 
+            'updated_at'
```

Comment:
Updated at and created at display the same time so we only need one. Also some of the column names should be renamed so a user could understand what the element is.

Reply:
Also the properties column only displays [object Object] rather then useful info

Reply:
I will make the activity log "pretty" for end users in a separate request. For now, this is the bare minimum to log whats going on with our models for debugging purposes.

Reply:
Okay sounds good merging.